redfish/logentrycollection.go

```diff
@@ -0,0 +1,59 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package redfish
+
+import (
+	"encoding/json"
+
+	"github.com/stmcginnis/gofish/common"
+)
+
+// LogEntry shall represent the log format for log services.
+type LogEntryCollection struct {
+	common.Entity
+
+	// ODataContext is the odata context.
+	ODataContext string `json:"@odata.context"`
+	// ODataType is the odata type.
+	ODataType string `json:"@odata.type"`
+	// Description provides a description of this resource.
+	Description string
+	// Members contain log entries of this log entry collection
+	Members []*LogEntry `json:"Members"`
+	// MembersCount is the number of log entries of this log entry collection.
+	MembersCount int `json:"Members@odata.count"`
+	// MembersNextLink is the link used for pagination
+	MembersNextLink string `json:"Members@odata.nextLink"`
+	// rawData holds the original serialized JSON so we can compare updates.
+	rawData []byte
+}
+
+// UnmarshalJSON unmarshals a LogEntry object from the raw JSON.
+func (logEntryCollection *LogEntryCollection) UnmarshalJSON(b []byte) (err error) {
```

Comment:
Since `rawData` isn't needed to be captured, this custom marshalling can be removed completely.